docs/tutorials/getting-started-with-flask.rst

```diff
@@ -0,0 +1,454 @@
+Build a rock for a Flask application
+------------------------------------
+
+In this tutorial you will create a simple Flask application and learn how to
+containerise it in a rock, using Rockcraft's ``flask-framework``
+:ref:`extension <flask-framework-reference>`.
+
+Setup
+=====
+
+We recommend starting from a clean Ubuntu 22.04 installation. If you don't have
+one available, you can create one using Multipass_:
+
+.. collapse:: How to create an Ubuntu 22.04 VM with Multipass
```

Comment:
I think the advantage of Multipass is that it can be installed in many platforms, like  Windows/mac/ other distribution. This is good, but clearly focused on another Ubuntu. Maybe at least a link/reference to the installation of Multipass?

Reply:
Sure. Added near the multipass installation.